clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/PTrans.java

```diff
@@ -60,7 +60,7 @@
  * @author Thomas Segismont
  */
 public class PTrans {
-    private static final Logger LOG = LoggerFactory.getLogger(PTrans.class);
+    private static final PTransLogger log = PTransLogging.getPTransLogger(PTrans.class);
```

Comment:
On 16 Sep 2015, at 13:11, Thomas Segismont wrote:

>> @@ -60,7 +60,7 @@
>> * @author Thomas Segismont
>> */
>> public class PTrans {
>> -    private static final Logger LOG = 
>> LoggerFactory.getLogger(PTrans.class);
>> +    private static final PTransLogger log = 
>> PTransLogging.getPTransLogger(PTrans.class);
>
> Not sure what you mean.

I was talking about the naming of that Interface class,
which is different from the pattern you use elsewhere.

-- 
hwr@pilhuhn.de  m:0179/207 4919  b:http://pilhuhn.blogspot.com